gazebo/rendering/LensFlare.cc

```diff
@@ -279,19 +283,41 @@ namespace gazebo
           pos.y /= pos.w;
           // check if light is visible
           if (std::fabs(pos.x) <= 1 &&
-              std::fabs(pos.y) <= 1 && pos.z > abs(pos.w))
+              std::fabs(pos.y) <= 1 && pos.z > -abs(pos.w))
```

Comment:
I know this check doesn't match the one done in WideAngleCamera.cc, but I think that was a bug noticed by Matias. I think @iche033 can confirm

Reply:
I don't see any difference in behavior in practice. If we set this back to positive, can someone please explain why? If it is inconsistent with the code in WideAngleCamera.cc it should have a comment explaining why.

Reply:
actually, looking at comparable code in [gz-rendering](https://github.com/gazebosim/gz-rendering/blob/27c186911b3043450bc429261b78a3fd1e9d71ea/ogre/src/OgreLensFlarePass.cc#L409-L414), it seems like maybe it should be as Terry has changed it here `z > -abs(w)`?

and should it be `fabs(w)`?